addons/mail/static/src/components/follow_button/follow_button.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.FollowButton" owl="1">
-        <t t-if="thread">
+        <t t-if="followButtonView and thread">
```

Comment:
Do you really need to check for the existence of `thread` since you already check for `followButtonView` that requires `thread` to exist?

Reply:
Yes, keep `thread` existence because it's not coupled with `followButtonView` yet. It's done in a following PR.

In the mean time, `thread` needs to be guarded with `followButtonView`.